Makefile

```diff
@@ -176,6 +176,8 @@ test-e2e-teardown:
 $(e2e_targets):: test-e2e-setup image/scorecard-test
 test-e2e:: $(e2e_tests) ## Run e2e tests
 
+test-e2e-sample-go:: dev-install cluster-create ## Run Memcached Operator Sample e2e tests
```

Comment:
Why create a separate target? Can't this be done along with test-e2e-go?
This is a part of testing scaffolded go memcached operator. Why not do it together?

Reply:
The target was created following the same structure as the others. See:

- test-e2e-go: it has the purpose to run the e2e test of the SDK project to provide projects that work and that the features provided by the CLI work well with ( scorecard, bundle, etc )
- test-e2e-sample-go: has the purpose of triggering the e2e tests scaffold only for the Memcached sample done with go/v3 

What was my motivation to create a separate target instead? 
- If the tests stop working/break is eaiser to identify what test has an issue that needs to be fixed as re-test locally only what is required to be tested ( faster ) 
- To run the test-e2e-go we need to build the scorecard images and etc which takes a long time when to run this one we just need to have a Kind up and running. They are lighter and much faster. 





I created a separate target so we are able to run only these checks if we need. IMO